submissions/ORybka/dom-js/main.js

```diff
@@ -0,0 +1,198 @@
+    mapPosition: ['scale(1.1)', '-40%', '-10%'],
+    dead: [1, 1, 1]
+  },
+  {
+    id: 8,
+    name: 'martell',
+    slogan: '"Unbowed, Unbent, Unbroken"',
+    description: 'House Martell of Sunspear is one of the Great Houses of Westeros. It rules the peninsula of Dorne in the far south of the continent from their castle Sunspear. Though loyal to the Iron Throne, the Martells were never conquered by the Targaryens and pursued a more isolated role in wider political events since Robert\'s Rebellion.',
+    mapPosition: ['scale(1.1)', '20%', '-20%'],
+    dead: [1, 1, 1]
+  },
+]
+
+
+let button;
+let buttonLi;
+let icons;
+let image;
+let imageCollection = [];
+let previousId = 0;
+const number = 3;
+const content = document.querySelector('.content');
+const house = document.querySelector('.house');
+const shield = document.querySelector('.shield');
+const slogan = document.querySelector('.slogan');
+const description = document.querySelector('.description');
+const background = document.querySelector('.content-container');
+const menu = document.querySelector(".menu-container");
+const navbar = document.querySelector(".navbar");
+const images = document.querySelector(".content-img-container");
+const hiddenIcon = document.querySelector(".icon");
+const header = document.querySelector(".header-container");
+
+function createList() {
+  houses.forEach(item => {
+    buttonLi = document.createElement('li');
+    button = document.createElement('button');
+    button.className = (`menu-item`);
+    button.innerHTML = `
+    <img class="menu-item-img" src="assets/shields/${item.name}.png"><span>House ${item.name}</span>`;
+    buttonLi.appendChild(button);
+    menu.appendChild(buttonLi);
+    button.addEventListener('click', () => {
+      createElements(item);
+      createImages(item);
+      animation(item);
+    })
+  })
+}
+
+function createElements(item) {
+  house.innerText = `house ${item.name}`;
+  shield.setAttribute('src', `assets/shields/${item.name}.png`);
+  slogan.innerText = `${item.slogan}`;
+  description.innerHTML = `<span>${item.description}</span>`;
+  background.style.backgroundColor = `var(--${item.name})`;
+}  
+
+function createImages(item) {
+  function imageCondition() {
+    for (let i = 0; i < number; i++) {
+      image = document.createElement('div');
+      image.className = ("content-img");
+      images.appendChild(image);
+      image.style.backgroundImage = `url("assets/members/member-${item.name}-${i}.jpg")`;
+      imageCollection.push(image);
+    }
+  }
+
+  if (images.children.length === 0) {
+    imageCondition()
+  } else {
+    images.innerHTML = ""; 
+    imageCondition()
+  }
+
+  if (imageCollection.length >= 3) {
+    imageHover(item);
+    imageCollection = [];
+  } else {
+    imageHover(item);
+  }
+}
+
+function animation(item) {
```

Comment:
It's better to start function name from a verb. And add more context to your variable names. 
For example, if you animate some card, you can add the name "animateCard". It would increase readability of your code for other developers